src/main/java/org/broadinstitute/hellbender/cmdline/ExomeStandardArgumentDefinitions.java

```diff
@@ -24,6 +24,9 @@
     public static final String SNP_FILE_LONG_NAME = "snpIntervals";
     public static final String SNP_FILE_SHORT_NAME = "SNP";
 
+    public static final String SITES_FILE_LONG_NAME = "siteIntervals";
+    public static final String SITES_FILE_SHORT_NAME = "SITES";
```

Comment:
I prefer `-SITES` vs. `-SNP` going forward, but left the old argument as it is used in the old CLIs.